Match arbitrary agent

benanhalt opened the pull request:
This branch modifies the Workbench uploader to choose an arbitrary agent when multiple agents are matched. The modification is just hardwired in with no control flag or anything. This was for experimental purposes.

benanhalt committed:
choose arbitrary match when uploading to agent table.

If multiple agent records match while uploading, one of the matching
agents will be chosen arbitrarily and treated as if it was an
unambiguous match.

maxpatiiuk committed:
Merge branch 'production' into match-arbitrary-agent

grantfitzsimmons commented:
This was extraordinarily helpful for my specific use case. I could not have done the conversion without it.

grantfitzsimmons committed:
Merge branch 'production' into match-arbitrary-agent

grantfitzsimmons committed:
Merge branch 'production' into match-arbitrary-agent

grantfitzsimmons committed:
Merge branch 'production' into match-arbitrary-agent

grantfitzsimmons committed:
Lint code with ESLint and Prettier

Triggered by 56716923ae15806ec05550476ef409d0ab911625 on branch refs/heads/match-arbitrary-agent

grantfitzsimmons committed:
Merge branch 'production' into match-arbitrary-agent
